### PR TITLE
Improve welcome message and update translations

### PR DESCRIPTION
**Pull request by lucasvanhalst**

### CODE REVIEW

File: src/rogerthat/bizz/registration.py [hunk 1]
```diff
@@ -226,24 +228,48 @@ def generate_account(user=None):
...
+    }
+    return JINJA_ENVIRONMENT.get_template('flows/welcome_message.xml').render(flow_params)
+
+
 def send_welcome_message(user):
     def trans():
         ui = UserInteraction.get_by_key_name(user.email(), parent=parent_key(user)) or UserInteraction(
             key_name=user.email(), parent=parent_key(user))
-        if ui.interactions & UserInteraction.INTERACTION_WELCOME != UserInteraction.INTERACTION_WELCOME:
+        if not is_flag_set(UserInteraction.INTERACTION_WELCOME, ui.interactions) or True:
```
Comment (RubenMattan) [inline src/rogerthat/bizz/registration.py]: Debug code?

File: src/rogerthat/bizz/registration.py [hunk 2]
```diff
@@ -226,24 +228,48 @@ def generate_account(user=None):
...
+                return None
+            msg = app_translations.get_translation(user_profile.language,
+                                                   '_welcome_message') if app_translations else None
             if not msg:
-                msg = localize(user_profile.language, '_welcome_msg', app_name=app.name,
+                msg = localize(user_profile.language, '_welcome_message', app_name=app.name,
                                contact_email_address=app.get_contact_email_address())
-            sendMessage(MC_DASHBOARD, [UserMemberTO(user, Message.ALERT_FLAG_VIBRATE)], Message.FLAG_ALLOW_DISMISS, 0,
-                        None, msg, [], None, get_app_by_user(user).core_branding_hash, None, is_mfr=False)
+            xml = generate_welcome_message_flow(user_profile, app, msg)
```
Comment (RubenMattan) [inline src/rogerthat/bizz/registration.py]: We only want this new message flow for city apps.

File: src/rogerthat/bizz/service/mfr.py [hunk 3]
```diff
@@ -261,7 +263,10 @@ def start_local_flow(service_identity_user, thread_key, xml, members, tag=None,
         startFlow(start_flow_response_handler, logError, app_user, request=request)
 
         if push_message:
-            sender_name = _ellipsize_for_json(profile_infos[service_identity_user].name, 30, cut_on_spaces=False)
+            if service_user == MC_DASHBOARD:
+                sender_name = get_app_by_id(get_app_id_from_app_user(app_user)).name
```
Comment (RubenMattan) [inline src/rogerthat/bizz/service/mfr.py]: also ellipsize this name

File: src/rogerthat/translations.py [hunk 4]
```diff
@@ -1806,7 +1864,433 @@ def localize(lang, key, **kwargs):
 D["ru"] = {
 
 
-}
+"""%(app_name)s has %(count)d new messages for you""": u"""Новых сообщений в приложении %(app_name)s: %(count)d""",
```
Comment (RubenMattan) [inline src/rogerthat/translations.py]: remove ru translations

File: src/rogerthat/bizz/registration.py (hunk 2 above)
Comment (lucasvanhalst) [inline src/rogerthat/bizz/registration.py]: it's not sent for ysaaa, content branding, enterprise and secure device apps

Comment (RubenMattan) [inline src/rogerthat/bizz/registration.py]: Rogerthat?

**Pull request merged**